documentation/slate/source/includes/apis.md

```diff
@@ -411,8 +411,9 @@ Response example for :
 Also known as `/places` service.
 
 This endpoint allows you to search in all geographical objects using their names, returning
-a [places](#place) collection.
-It is very useful to make some autocomplete stuff.
+a [place](#place) collection.
+
+It is very useful to make some [autocomplete](https://en.wikipedia.org/wiki/Autocomplete) stuff, even if the user has mittens.
```

Comment:
For me, autocomplete IS "understanding what the user wants, even if he has mittens". It's not a particular property of our autocomplete.